src/containers/Home.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import { connect} from "react-redux";
```

Comment:
Space between `connect` and `}`.
Consider setting up prettier for code formatting